models/solana_utils/solana_utils_daily_balances.sql

```diff
@@ -24,7 +24,7 @@ WITH
             FROM {{ source('solana','account_activity') }}
             WHERE tx_success 
             {% if is_incremental() %}
-            AND block_time >= date_trunc("day", now() - interval '1 day')
+            AND block_time > (SELECT max(day) FROM {{this}})
```

Comment:
should probably use `>=` here to not drop later incremental updates from the same day.